ui/src/app/cohort-review/detail-tab-table/detail-tab-table.component.ts

```diff
@@ -4,14 +4,8 @@ import {ClrDatagridComparatorInterface} from '@clr/angular';
 import {Subscription} from 'rxjs/Subscription';
 
 import {currentCohortStore, currentWorkspaceStore} from 'app/utils/navigation';
-import {CohortReviewService, PageFilterRequest, ParticipantData, SortOrder} from 'generated';
+import {CohortReviewService, PageFilterRequest, SortOrder} from 'generated';
 
-class SortByColumn implements ClrDatagridComparatorInterface<ParticipantData> {
-  compare(a: ParticipantData, b: ParticipantData) {
-    return (a.numMentions === null ? 0 : parseInt(a.numMentions, 10))
-        - (b.numMentions === null ? 0 : parseInt(b.numMentions, 10));
-  }
-}
```

Comment:
This interface was converting `numMentions` to an int so it would sort as a number. With it removed, it now sorts a string. Was this intended?

Reply:
Done.